cluster-autoscaler/integration/framework.go

```diff
@@ -101,13 +105,20 @@ func (driver *Driver) adjustNodeGroups() error {
 		return err
 	}
 
-	for index, machineDeployment := range machineDeployments.Items {
+	firstMDOfWorkerWithThreeZones := true
```

Comment:
for better readability
```suggestion
	isFirstMdOfWorkerWithThreeZones := true
```